### ISSUE TITLE
ZeroDivisionError in cpu_linux.py

### ISSUE THREAD

**Issue by ivasic** [elastic/apm-agent-python#610]
**Describe the bug**:

When deploying my Django app on AWS Beanstalk  the APM agent throws `ZeroDivisionError` with the trace: 
``` python
ZeroDivisionError: float division by zero
  File "elasticapm/utils/threading.py", line 79, in run
    rval = self._function(*self._args, **self._kwargs)
  File "elasticapm/metrics/base_metrics.py", line 97, in collect
    for data in metricset.collect():
  File "elasticapm/metrics/base_metrics.py", line 195, in collect
    self.before_collect()
  File "elasticapm/metrics/sets/cpu_linux.py", line 72, in before_collect
    cpu_usage_ratio = delta["cpu_usage"] / delta["cpu_total"]
```

**To Reproduce**

I'm not able to consistently reproduce this behavior but the problem obviously seems that the `cpu_total` read from the stats file is zero. It may be due to the newly launched docker container that has just started up.

One way to reproduce this (although in my local tests it rarely happens):
- Launch a new `python:3.7` container
- Run any django `manage.py` command as `CMD` via Dockerfile 

**Expected behavior**:

No `ZeroDivisionError` exception is thrown

**Environment (please complete the following information)**
- OS: Linux
- Python version: 3.7.4 (happens on 3.7.3 as well)
- Framework and version [e.g. Django 2.1]:  Django 2.2.6
- APM Server version: v7.4.0
- Agent version: 5.2.1

**Additional context**

I managed to fix this by catching this exception and setting the ration to 0 which I think makes sense. 

``` python
try:
    cpu_usage_ratio = delta["cpu_usage"] / delta["cpu_total"]
except ZeroDivisionError:
    cpu_usage_ratio = 0
```

Similar problem happens with `cpu_process_percent` which I solved the same way.
I can open up a pull request (with a corresponding test case in `cpu_linux_test.py`?) if that makes it easier.

Thanks,
Ivan

**Comment by beniwohli** [comment]
Hi @ivasic 

great catch, thanks for opening the issue! If you're up for it, a pull request would be great! The approach you outlined above looks sensible to me.

**Comment by basepi** [comment]
Fixed in #611
